### PR TITLE
Added a heredoc example for Style/MutableConstant

### LINKED ISSUE #5725: Style/MutableConstant on heredocs
## Expected behavior

Rubocop errors about style/MutableConstant for heredocs

## Actual behavior

Either it should not error, or there should be documentation on how to handle that case

## Steps to reproduce the problem

With the default rubocop config, run the linter on the following:

```rb
TESTING = <<TESTING
  content
TESTING
```

## RuboCop version

Include the output of `rubocop -V`. Here's an example:

```
$ rubocop -V
0.52.1
```

### PR DESCRIPTION
**Pull request by dkniffin**
Resolves #5725
-----------------

Before submitting the PR make sure the following are checked:

* [x] Wrote [good commit messages][1].
* [x] Commit message starts with `[Fix #issue-number]` (if the related issue exists).
* [x] Feature branch is up-to-date with `master` (if not - rebase it).
* [x] Squashed related commits together.
* [x] Added tests.
* [x] Added an entry to the [Changelog](../blob/master/CHANGELOG.md) if the new code introduces user-observable changes. See [changelog entry format](../blob/master/CONTRIBUTING.md#changelog-entry-format).
* [x] The PR relates to *only* one subject with a clear title
  and description in grammatically correct, complete sentences.
* [x] Run `rake default` or `rake parallel`. It executes all tests and RuboCop for itself, and generates the documentation.

[1]: http://chris.beams.io/posts/git-commit/

### CODE REVIEW

**Comment by pocke** [comment]
`manual/cops_style.md` is generated from comments in the source code automatically. So can you edit the  comment and run `rake generate_cops_documentation`?
https://github.com/bbatsov/rubocop/blob/master/lib/rubocop/cop/style/mutable_constant.rb#L9-L14

**Comment by dkniffin** [comment]
@pocke Done.

File: lib/rubocop/cop/style/mutable_constant.rb [hunk 1]
```diff
@@ -12,6 +12,11 @@ module Style
       #
       #   # good
       #   CONST = [1, 2, 3].freeze
+      #
+      #   # good
+      #   CONST = <~TESTING.freeze
```
Comment (pocke) [inline lib/rubocop/cop/style/mutable_constant.rb]: It has a syntax error, could you replace `<~` with `<<~` and re-run `rake generate_cops_documentation`?

**Comment by mikegee** [comment]
Hey @dkniffin, it looks like you pushed but the change isn't in the commit? 🤔

**Comment by dkniffin** [comment]
🤦‍♂️ It's up now.

**Comment by bbatsov** [comment]
👍

**Pull request merged**